evaluator/src/main/kotlin/PackageRule.kt

```diff
@@ -136,7 +136,7 @@ open class PackageRule(
      */
```

Comment:
Commit message: The body text is a bit strange, there seems to be a preposition too much. Maybe change to something like "Rename the method to make a clear distinction between repository-wide and package-local license choices." ?